mne/beamformer/_lcmv.py

```diff
@@ -166,6 +179,9 @@ def make_lcmv(info, forward, data_cov, reg=0.05, noise_cov=None, label=None,
         will be computed (Borgiotti-Kaplan beamformer) [2]_,
         if 'nai', the Neural Activity Index [1]_ will be computed,
         if None, the unit-gain LCMV beamformer [2]_ will be computed.
+    reduce_rank : bool | int
+        If specified, the rank of the leadfield will be reduced. True will
+        reduce the rank by 1. The rank can also be specified as integer.
```

Comment:
add:

```
Setting reduce_rank to True is typically necessary if you use a single sphere model for MEG.
```

Reply:
Maybe I don't understand the subtlety but can't we detect this automatically numerically?

Reply:
@sarangnemo says it will be fragile. I think we have now a clear enough error message